src/brevitas_examples/imagenet_classification/ptq/ptq_evaluate.py

```diff
@@ -332,7 +363,12 @@ def main():
             model,
             equalize_iters=args.graph_eq_iterations,
             equalize_merge_bias=args.graph_eq_merge_bias,
-            merge_bn=not args.calibrate_bn)
+            merge_bn=not args.calibrate_bn,
```

Comment:
can also change this to merge_bn or remove that flag from argparse

Reply:
Let's change it